.github/workflows/build.yml

```diff
@@ -86,3 +86,8 @@ jobs:
         node-version: '16.x'
         java-version: '11'
         vtag: ${{ env.vtag }}
+    - name: Generate Sponsors
```

Comment:
After giving this some thought, the list of sponsors has nothing to do with the build. I suggest creating a new workflow that uses cron to automatically regen the list of sponsors every 12 or 24 hours using https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule.

Reply:
Will do! Just want to see if the general gen works, then we can check the best place.

Reply:
@cb1kenobi Updated!